### PR TITLE
Update contributors

### PR DESCRIPTION
**Pull request by clementbiron**
Hi @Harshil-Jani 

I am adding you to the list of contributors of this repository consolidated in the `.all-contributorsrc` file so that I can automatically list you on Open Terms Archive [about page](https://opentermsarchive.org/en/about/). The information used is the public information (name, login, avatar and profile URL) from your GitHub account. If you wish to correct any information or object to this addition, please let me know.

Thank you 🙏

@MattiSG that should be review and merge after https://github.com/OpenTermsArchive/deployment/pull/49

### CODE REVIEW

File: .all-contributorsrc [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+      "avatar_url": "https://avatars.githubusercontent.com/u/222463?v=4",
+      "profile": "https://mattischneider.fr/",
+      "contributions": [
+        "code",
+        "doc"
+      ]
+    },
+    {
+      "login": "martinratinaud",
+      "name": "Martin Ratinaud. Freelancer.",
```
Comment (MattiSG) [inline .all-contributorsrc:30]:
```suggestion
      "name": "Martin Ratinaud",
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -52,15 +52,15 @@ decisions when appropriate.
...
+Examples of representing our community include using an official email address,
 posting via an official social media account, or acting as an appointed
 representative at an online or offline event.
 
 ## Enforcement
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the community leaders responsible for enforcement at
-moderators@opentermsarchive.org.
+[INSERT CONTACT METHOD].
```
Comment (MattiSG) [inline CODE_OF_CONDUCT.md:63]: Nope 😉

**Review by MattiSG — CHANGES_REQUESTED** [review]
Blocking: removal of enforcement address.

File: .all-contributorsrc [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      ]
+    },
+    {
+      "login": "Harshil-Jani",
+      "name": "Harshil Jani",
+      "avatar_url": "https://avatars.githubusercontent.com/u/79367883?v=4",
+      "profile": "https://github.com/Harshil-Jani",
+      "contributions": [
+        "code",
+        "doc"
```
Comment (MattiSG) [inline .all-contributorsrc:45]:
```suggestion
        "code"
```

**Review by MattiSG — APPROVED** [review]

**Comment by clementbiron** [comment]
In order to deliver, I'm taking responsibility for bypassing the changelog test.

**Pull request merged**